server/datastore/mysql/labels.go

```diff
@@ -695,7 +695,7 @@ func (ds *Datastore) applyHostLabelFilters(ctx context.Context, filter fleet.Tea
 			return "", nil, ctxerr.Wrap(ctx, err, "get software installer metadata by team and title id")
 
 		default:
-			// TODO(uniq): prior code was joining on installer id but based on how list options are parsed [1] it seems like this should be the title id
+			// TODO(Sarah): prior code was joining on installer id but based on how list options are parsed [1] it seems like this should be the title id
```

Comment:
@gillespi314 I changed the TODO to you since it's not related to the unified queue, let me know if that's ok with you (or if you want me to just remove it)?

Reply:
On second thought, is it really fixed or is your change the actual fix? I.e. is it just to raise awareness that this was changed, or does that current implementation need to be changed? I'd remove the comment altogether if there's not really anything to fix.

Reply:
@mna, you're right, it was more so to raise awareness of the change in case I might have missed something. 

@jahzielv, do you mind giving this a quick sanity check? There's a similar note regarding VPP to check [here](https://github.com/fleetdm/fleet/pull/26067/files#diff-ff52357cf22cef1a99b8ecd5a45c146be49a3e58f63c06845ae4ef3879a7e014R1311).

Reply:
@gillespi314 so sorry, I totally missed this notification! I can take a look